tests/integration/replication.tcl

```diff
@@ -316,15 +316,9 @@ foreach mdl {no yes} {
                             stop_write_load $load_handle3
                             stop_write_load $load_handle4
 
-                            # Make sure that slaves and master have same
-                            # number of keys
-                            wait_for_condition 500 100 {
-                                [$master dbsize] == [[lindex $slaves 0] dbsize] &&
-                                [$master dbsize] == [[lindex $slaves 1] dbsize] &&
-                                [$master dbsize] == [[lindex $slaves 2] dbsize]
-                            } else {
-                                fail "Different number of keys between master and replica after too long time."
-                            }
+                            wait_for_ofs_sync $master [lindex $slaves 0]
```

Comment:
i think we need a call to the new `wait_load_handlers_disconnected` here.
otherwise, in theory, we can see that the master ans replica are in sync, but there are still some commands in the pipeline

Reply:
added